package.json

```diff
@@ -10,10 +10,12 @@
   "author": "mojisola aramide",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^5.1.1",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "nodemon": "^3.0.1",
```

Comment:
Nodemon Is a Dev Dependency

Reply:
I dont understand this, i use it to start index.js

Reply:
Yes But Its For Development Purpose